conf/messages.cy

```diff
@@ -457,21 +457,25 @@ your.account.your.groups.off.p=Mae’r grwpiau mynediad wedi’u hanalluogi. Dim
 your.account.your.groups.p=Nid ydych wedi’ch neilltuo i unrhyw grŵp mynediad ar hyn o bryd
 
 # View/update contact details
-update-contact-details.title=Contact details TBC
-update-contact-details.summary-list.changes-are-pending.title=Important TBC
-update-contact-details.summary-list.changes-are-pending=You asked us to change these details on {0}. We aim to make these changes within 28 days. TBC
-update-contact-details.summary-list.body.p1=To update these contact details, you must be one of the following: TBC
-update-contact-details.summary-list.body.li1=a director TBC
-update-contact-details.summary-list.body.li2=a proprietor TBC
-update-contact-details.summary-list.body.li3=a company secretary TBC
-update-contact-details.summary-list.body.li4=a partner TBC
-update-contact-details.summary-list.body.li5=a sole trader TBC
-update-contact-details.summary-list.body.p2=We aim to update your contact details within 28 days. We’ll send you a letter when this is done. TBC
-update-contact-details.summary-list.email=Email address TBC
-update-contact-details.summary-list.phone=Telephone number TBC
-update-contact-details.summary-list.name=Business name TBC
-update-contact-details.summary-list.address=Business address TBC
-update-contact-details.summary-list.none=None TBC
+view-contact-details.title=Manylion cyswllt
+view-contact-details.heading=Manylion cyswllt
```

Comment:
how come this is a duplicate key? why not just use one key for both heading & title

Reply:
Oh that's a habit from previous teams, we sometimes had situations where they were different slightly. Happy to change it

Reply:
Ahh, yea I can think of some cases where they'd need to be slightly different. But if it's the exact same I'd rather not have a duplicate. In general h1 & title should be the same (minus the service name - GOV.UK suffix) so using the same key where possible helps with that.